packages/ethereum_api/lib/src/wallet/wallet_api_client/web.dart

```diff
@@ -186,8 +205,11 @@ class EthereumWalletApiClient implements WalletApiClient {
   /// - [UnknownWalletFailure]
   @override
   Future<WalletCredentials> getWalletCredentials() async {
-    _checkWalletAvailability();
+    final isEthereumBrowserAvailable = _checkEthereumAvailability();
     try {
+      if (isEthereumBrowserAvailable == false) {
```

Comment:
`if (!isEthereumBrowserAvailable)`

Reply:
From my understanding, the stub is used for when the dart library compiles natively for mobile vs web, but since I am adding to my home screen from a web app, the app does not actually compile down to mobile, it's still a website.  

Using the stub won't fix metamask dependency is my understanding.

I can write out the mobile version of the api client though, but that won't fix the current problem.